2017-06-03 11:45:57,281 INFO monitors.py [line:126]    "quorum_leader_name": "ubuntu-A",
stdin: is not a tty

2017-06-03 11:45:57,281 INFO monitors.py [line:129]    "quorum_leader_name": "ubuntu-A",
2017-06-03 11:45:57,281 INFO initConfig.py [line:18] Init osd process
2017-06-03 11:45:59,686 ERROR node.py [line:55] tdin: is not a tty
pool 'reliablityTestPool' already exists

2017-06-03 11:45:59,686 ERROR node.py [line:57] pool reliablityTestPool created failed
2017-06-03 11:45:59,686 ERROR node.py [line:58] Error message is tdin: is not a tty
pool 'reliablityTestPool' already exists

